docs/pages/play.vue

```diff
@@ -36,6 +36,24 @@
             <input type="hidden" name="js_wrap" value="l">
             <b-btn size="sm" type="submit" :disabled="!isOk">Export to JSFiddle</b-btn>
           </form>
+          <form
```

Comment:
The header section markup isn't that good. We should improve that in a follow-up PR...

Reply:
Yeah, as long as it looks good at different viewport breakpoints.

Reply:
Done.